chain/chunks/src/lib.rs

```diff
@@ -1477,9 +1483,6 @@ impl ShardsManager {
                 parts: forwarded_parts.into_iter().map(|(_, part)| part).collect(),
                 receipts: Vec::new(),
             };
-            // Make sure we mark this chunk as being requested so that it is handled
-            // properly in the next call. Note no requests are actually sent at this point.
-            self.request_chunk_single_mark_only(header);
```

Comment:
This line was for by-passing the check about `chunk_requested` at line 1409 for the following call of `self.process_partial_encoded_chunk`.